Simplenote/NoteEditorViewController.m

```diff
@@ -1160,4 +1147,25 @@ - (NSPopover *)newPopoverWithContentViewController:(NSViewController *)viewContr
     return popover;
 }
 
+- (BOOL)urlSchemeIsAllowed: (NSString *) scheme {
+    return [scheme isEqualToString:@"http"] ||
+        [scheme isEqualToString:@"https"] ||
+        [scheme isEqualToString:@"mailto"];
+}
+
+#pragma mark - WKNavigationDelegate
+
+- (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler {
+    if (navigationAction.navigationType == WKNavigationTypeLinkActivated) {
+        NSURL *linkUrl = navigationAction.request.URL;
+        if ([self urlSchemeIsAllowed:linkUrl.scheme]) {
```

Comment:
It'd be nice if we could allow anchor links, too (within the current document) – useful for tables of content.

Reply:
> It'd be nice if we could allow anchor links, too

Darn, I couldn't get this to work. When you do `[click me](#myanchor)` macOS thinks the link is a file url, which is not so good security-wise.